pkgs/build-support/setup-hooks/multiple-outputs.sh

```diff
@@ -154,6 +154,16 @@ _multioutDocs() {
 
     moveToOutput share/info "${!outputInfo}"
     moveToOutput share/doc "${!outputDoc}"
+
+    # move gi-docgen docs to devdoc
+    for output in $(getAllOutputNames); do
+        if [ -d "${!output}/share/doc" ]; then
+            while IFS= read -r -d '' dir; do
+                moveToOutput "share/doc/$(basename "$dir")" "${!outputDevdoc}"
```

Comment:
IIRC, I have seen docs installed to subdirectories so `basename` may not work.

Reply:
so something like this then

```sh
$ export output="/nix/store/something"
$ export dir="/nix/store/something/share/doc/ok"
$ dirname "${dir#"$output/"}"
share/doc